templates/icinga.ini

```diff
@@ -298,5 +298,276 @@ min = "0"
+
+
+[icinga-current-concurrent-checks.graph]
+check_command = "icinga"
+
+[icinga-current-concurrent-checks.metrics_filters]
+checks.value = "$service_name_template$.perfdata.current_concurrent_checks.value"
+
+[icinga-current-concurrent-checks.urlparams] 
+title = "Current Concurrent Checks"
+areaAlpha = "0.5"
+areaMode = "all"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+    
+[icinga-current-concurrent-checks.functions]
+checks.value = "alias(color($metric$, '#61aaed'), 'Checks')"
+
+
+[icinga-last-messages-received.graph]
+check_command = "icinga"
+
+[icinga-last-messages-received.metrics_filters]
+sent.value = "$service_name_template$.perfdata.last_messages_sent.value"
+received.value = "$service_name_template$.perfdata.last_messages_received.value"
+
+[icinga-last-messages-received.urlparams]         
+title = "Last Messages Sent/Received"
+areaMode = "all"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+    
+[icinga-last-messages-received.functions]
+sent.value = "alias(color($metric$, '#61aaed'), 'Sent')"
+received.value = "alias(color($metric$, '#1a7dd7'), 'Received')"
+
+
+[icinga-passive-service-checks.graph]
+check_command = "icinga"
+
+[icinga-passive-service-checks.metrics_filters]
+1min.value = "$service_name_template$.perfdata.passive_service_checks_1min.value"
+5min.value = "$service_name_template$.perfdata.passive_service_checks_5min.value"
+15min.value = "$service_name_template$.perfdata.passive_service_checks_15min.value"
+
+[icinga-passive-service-checks.urlparams]
+title = "Passive Service Checks"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-passive-service-checks.functions]
+1min.value = "alias(color($metric$, '#61aaed'), 'Checks/1m')"
+5min.value = "alias(color($metric$, '#1a7dd7'), 'Checks/5m')"
+15min.value = "alias(color($metric$, '#0b3c68'), 'Checks/15m')"
+
+
+[icinga-remote_check_queue.graph]
+check_command = "icinga"
+
+[icinga-remote_check_queue.metrics_filters]
+queue.value = "$service_name_template$.perfdata.remote_check_queue.value"
+
+[icinga-remote_check_queue.urlparams]
+title = "Remote Check Queue"
+areaMode = "none"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-remote_check_queue.functions]
+queue.value = "alias(color($metric$, '#61aaed'), 'Checks')"
+
+
+[icinga-sum-bytes-sent-per-second.graph]
+check_command = "icinga"
+
+[icinga-sum-bytes-sent-per-second.metrics_filters]
+sent_per_second.value = "$service_name_template$.perfdata.sum_bytes_sent_per_second.value"
+received_per_second.value = "$service_name_template$.perfdata.sum_bytes_received_per_second.value"
+
+[icinga-sum-bytes-sent-per-second.urlparams]
+title = "Sum Bytes Sent/Received"
+areaMode = "all"
+lineWidth = "2"
+min = "0"
+yUnitSystem = "none"
+
+[icinga-sum-bytes-sent-per-second.functions]
+sent_per_second.value = "alias(color($metric$, '#61aaed'), 'Sent/s')"
+received_per_second.value = "alias(color($metric$, '#1a7dd7'), 'Received/s')"
```

Comment:
1. I'd change the legends to *Subject (Unit)* if there's a non-redundand subject, i.e. "Sent (bytes/s)".
2. Because these are bytes, you should set the respective *yUnitSystem*.